src/TodoInputField.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import './index.scss';
+
+class TodoInputField extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      text: '',
+    };
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+  }
+  handleChange(e) {
+    this.setState({ text: e.target.value });
+  };
+
+  handleSubmit(e) {
```

Comment:
This is an interesting approach. Normally I would wait for the user to be done typing and have them physically hit "Enter" or click "Submit" for this to be fired.